src/sentry/analytics/event.py

```diff
@@ -101,7 +104,8 @@ def __init__(self, type=None, datetime=None, **items):
     def serialize(self):
         return dict(
             {
-                'timestamp': int(self.datetime.isoformat('%s')),
+                'guid': self.guid,
+                'timestamp': int(self.datetime.strftime('%s')),
```

Comment:
Shouldn't this use `sentry.utils.dates.to_timestamp`?

I'm not sure if this was ever resolved: https://bugs.python.org/issue12750#msg142249

Reply:
TIL

I can switch it to this.

Reply:
Done in c406b2763cf764c562913fc2e5187063a039ce37